marshmallow_jsonapi/validate.py

```diff
@@ -0,0 +1,25 @@
+from marshmallow.validate import Validator, ValidationError
+
+class ForeignKey(Validator):
```

Comment:
I'm not sure this is within scope; JSON API only requires that `id` values are strings; they need not be integers.